src/Widgets/WindowSwitcher.vala

```diff
@@ -173,6 +182,27 @@ namespace Gala
 			dock_surface = null;
 		}
 
+		void update_background ()
```

Comment:
I can't verify if `Meta.Screen.get_size ()` works for this case since I don't have multiple monitors. Looking into libmutter's documentation also does not give any explanation what this method exactly returns. It seems that it's returning the size only for one monitor, not all of them combined.